src/Hedgehog/Gen.fs

```diff
@@ -4,16 +4,41 @@ open System
 
 /// A generator for values and shrink trees of type 'a.
 [<Struct>]
-type Gen<'a> =
-    | Gen of Random<Tree<'a>>
+type Gen<'a> = private {
+    Config : GenConfig<'a>
+    Random : Random<Tree<'a>>
+}
 
 module Gen =
+    module Config =
```

Comment:
`PropertyConfig` exists. In the interest of consistency, I think we should have this either be `GenConfig` or change `PropertyConfig` to `Property.Config`. I _believe_ [when I wrote the original](https://github.com/hedgehogqa/fsharp-hedgehog/pull/288) `PropertyConfig`, I was simply copying what Haskell did. I don't feel strongly in either direction.